### PR TITLE
vpnc: update to maintained github fork

### LINKED ISSUE #14119: vpnc: build failure: svn SSL certificate verification failed
Maintainer: @danielg4
Environment: ipq806x, OpenWRT git master

Description:

When trying to build OpenWRT, I get the following error:

`logs/package/feeds/packages/vpnc/compile.txt`:
```
Checking out files from the svn repository...
Redirecting to URL 'https://svn.unix-ag.uni-kl.de/vpnc/trunk':
svn: E170013: Unable to connect to a repository at URL 'https://svn.unix-ag.uni-kl.de/vpnc/trunk'
svn: E230001: Server SSL certificate verification failed: issuer is not trusted
make[3]: *** [Makefile:95: /home/amezin/openwrt/dl/vpnc-0.5.3.r550.tar.gz] Error 1
time: package/feeds/packages/vpnc/compile#0.15#0.09#1.70
```

I've got around the issue by manually performing `svn co https://svn.unix-ag.uni-kl.de/vpnc/trunk` and permanently accepting the certificate

### PR DESCRIPTION
**Pull request by lipnitsk**
The SVN-based version has not changed in years. Many distros use this
fork as evident here: https://github.com/streambinder/vpnc/issues/14

Compile tested against GnuTLS and OpenSSL on ramips target.

Fixes #14119.

Signed-off-by: Ilya Lipnitskiy <ilya.lipnitskiy@gmail.com>

Maintainer: @danielg4 
Compile tested: ramips, mt7621, master

### CODE REVIEW

**Comment by danielg4** [comment]
As I previously mentioned, I can no longer test these changes, but they look correct to me, albeit the splitting of line 39 into multiple lines seems gratuitous. ACK.

**Comment by lipnitsk** [comment]
Yeah, still to be tested, but this is the direction we need to move to. Looking to the OpenWrt community to help test master or once the release branch is created. Thanks for quick response.

**Comment by danielg4** [comment]
I'm sure it could be backported to existing releases also.

File: net/vpnc/Makefile [hunk 1]
```diff
@@ -8,19 +8,17 @@
...
-PKG_SOURCE_VERSION:=$(PKG_REV)
-PKG_SOURCE_PROTO:=svn
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/streambinder/vpnc
+PKG_MIRROR_HASH:=df833bbe369bb96cb915da9b63e4dded0f676f06bcdada4ef94e56b8d87b187e
 
 PKG_MAINTAINER:=Daniel Gimpelevich <daniel@gimpelevich.san-francisco.ca.us>
-PKG_LICENSE:=VARIOUS
-PKG_LICENSE_FILES:=COPYING
+PKG_LICENSE:=GPL-3.0
```
Comment (neheb) [inline net/vpnc/Makefile:20]: Should be -or-later

**Comment by dwmw2** [comment]
Does anyone still care about vpnc? I'd quite like to implement the IPSec support in OpenConnect and obsolete it completely if so...

**Comment by danielg4** [comment]
People do care, and I mentioned OpenConnect in the fixed issue…

**Pull request merged**